Associative-Containers/exercise2.cpp

```diff
@@ -1,85 +1,141 @@
-#include<map>
-#include<iostream>
-#include<numeric>
-#include<cmath>
-#include<functional>
-#include<cassert>
+#include <map>
+#include <iostream>
+#include <numeric>
+#include <cmath>
+#include <functional>
+#include <cassert>
+#include <iomanip>
+enum class ErrorCode
+{
 
-enum class ErrorCode{
-    
     Ok,
     BadCharacter,
     BadFormat,
     DivideBy0,
     SqrtOfNegativeNumber,
     ModuleOfNonIntegerValue,
-    None 
-    };
+    None
+};
+bool DivideBy0 (const std::string &input_)
+{
+    for(size_t i = 0;i<input_.size();i++)
```

Comment:
Be consequent with the writing style. 
Please add spaces around **<**operator.
Please add a space after ';'
```suggestion
    for(size_t i = 0; i < input_.size(); i++)
```

Reply:
Uploaded